docs/src/query/operators.md

```diff
@@ -218,6 +219,29 @@ The examples below will use the table definitions in [table tiers](../reproduce/
 
 <!-- ## Join appears here in the general docs -->
 
+## Top
+
+Similar to the univeral set operator, the top operator uses `dj.Top` notation. It is used to 
+restrict a query by the given `limit`, `order_by`, and `offset` parameters:
+
```

Comment:
```
| Parameter | Keyword Arguments | Default Argument | Description |
| -- | -- | -- | -- |
| `limit` | | 1 | Restrict the number of rows returned. |
| `order_by` | "KEY", "DESC", "ASC" | "KEY" |  |
| `offset` | | 0  |  |
```

Reply:
Please fill out the rest of the table.  And perhaps the headings I am using (`Keyword Arguments`, `Default Argument`) are not the clearest, so feel free to replace.